shared/templates/template_BASH_regex_permissions

```diff
@@ -0,0 +1,6 @@
+# platform = multi_platform_all, Red Hat Enterprise Linux 5
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+find FILEPATH -regex '^FILEPATH/FILENAME' -exec chmod FILEMODE {} \;
```

Comment:
I've checked the oval definition. The operation="pattern match" supports only regex syntax, which means I can't write file names using globbing without translate the syntax from globbing to regex before generating the oval file. It looks complicated...